examples/status_msg.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+'''
+Print flight controller banner statustext messages
+
+SPDX-FileCopyrightText: 2024 Amilcar do Carmo Lucas <amilcar.lucas@iav.de>
+
+SPDX-License-Identifier: GPL-3.0-or-later
+'''
+
+import time
+
+from argparse import ArgumentParser
+
+from pymavlink import mavutil
+
+
+def print_banner_messages(master):
+    '''Decode and print banner information from the flight controller'''
```

Comment:
I'm a bit concerned by this "decode" bit here.  Statustexts really should be opaque to software, they're really not a good way of transmitting data to automated systems.

An example decoding the `AUTOPILOT_VERSION` message would be good, 'though :-)  Particularly because of the way we encode the board ID in that message.
```suggestion
    '''print banner information from the flight controller'''
```